src/util/logger-utils.ts

```diff
@@ -0,0 +1,72 @@
+import { Writable } from "stream";
+import safeJsonStringify from "safe-json-stringify";
+import bformat from "bunyan-format";
+import { createHashWithSharedSecret } from "utils/encryption";
+
+const SENSITIVE_DATA_FIELDS = ["jiraHost", "orgName", "repoName", "userGroup", "userGroup", "aaid", "username"];
+// For any Micros env we want the logs to be in JSON format.
+// Otherwise, if local development, we want human readable logs.
+const outputMode = process.env.MICROS_ENV ? "json" : "short";
+
+//TODO Remove this code when there will be convenient way to do it in Probot.
+//  See https://github.com/probot/probot/issues/1577
+export const filterHttpRequests = (record: Record<string, any>, filteredLoggerName: string) => {
+	const { msg, name } = record;
+	if (name !== filteredLoggerName) {
+		return false;
+	}
+	return !!msg.match(/(GET|POST|DELETE|PUT|PATCH)/);
+};
+
+export class RawLogStream extends Writable {
+	private readonly  filteredHttpLoggerName: string;
+	private readonly isUnsafeStream: boolean;
+	private writeStream: NodeJS.WritableStream;
+
+	public constructor(filteredHttpLoggerName: string, isUnsafeStream = false) {
+		super({ objectMode: true });
+		this.filteredHttpLoggerName = filteredHttpLoggerName;
+		this.isUnsafeStream = isUnsafeStream;
+		this.writeStream = bformat({ outputMode, levelInString: true });
+	}
+
+	public async _write(record: any, encoding: BufferEncoding, next): Promise<void> {
+		const { booleanFlag, BooleanFlags } = await import("config/feature-flags");
```

Comment:
Not a huge fan of importing this here, but it was the lesser of two evils.
Since feature flags depends on logger and logger on feature flags now... I can't import it up top since they depend on each other.

I can replace feature-flags logger with a standard bunyan logger... which will allow it to run fine and dandy, but it starts complicating tests. Am trying a few things still.

Reply:
Agree that `import()` is not great. I agree having a standard bunyan logger just inside the feature-flag. 
Maybe we don't have to test the logging part within the feature flags, so it is be easier?

Reply:
I'll have another crack at it and see what I can get.

The logger tests are fine, it breaks the featureflag tests because it tries to access it before its ready but I should be able to wait for launchdarkly... we shall see!!!

Reply:
Why are we using  a FF here at all?  This is a non-functional change and causes circular dependencies if we add FF.  Seems like a good argument to not have it behind an FF.  Plus, we want to start obfuscating the logs ASAP.